src/script/page/RightSidebar/RightSidebar.tsx

```diff
@@ -166,6 +166,18 @@ const RightSidebar: FC<RightSidebarProps> = ({
     });
   }, []);
 
+  const containerRef = useCallback(
+    (element: HTMLDivElement | null) => {
+      const focussableElements =
+        'a:not([disabled]), button:not([disabled]), input[type=text]:not([disabled]), [tabindex]:not([disabled]):not([tabindex="-1"])';
+      const nextElementToFocus = element?.querySelectorAll(focussableElements)[0] as HTMLElement;
```

Comment:
Damn, would have taken forever to figure out :D :+1:

Reply:
Found it on https://stackoverflow.com/a/35173443/7091035